eng/PatchConfig.props

```diff
@@ -35,5 +35,10 @@ Later on, this will be checked using this condition:
       Microsoft.AspNetCore.SignalR.Redis;
     </PackagesInPatch>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(VersionPrefix)' == '2.1.12' ">
+    <PackagesInPatch>
+      Microsoft.AspNetCore.Mvc.Core;
```

Comment:
@aspnet/build please double-check this when you can :).

Reply:
PatchConfig.props looks good now.